_posts/2016-10-10-episerver-developer-meetup-solita.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Episerver developer meetup at Solita
+author: Rinorragi
+date: 2016-10-10 14:00:00 +0200
+excerpt: What happened in the second Episerver developer meetup held in Helsinki
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- security
+---
+Solita organized a meetup for Finnish Episerver developers in Helsinki at 29.9. Meetup consisted of food, beer and presentations. It was second of a kind and as fun and cozy as its successor. This time we had a bit more time in the end for the networking than last time!
```

Comment:
"The event was a successor to the spring's Episerver dev meetup and was as fun and cozy as the first one...?"